content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa no pc do usuário ou em um pod
+    - proxies de um host local endereçam para o servidor de API do kubernetes
+    - cliente para o proxi usa HTTP 
+    - proxi para o servidor de API usa HTTPS 
+    - localiza o servidor de API
+    - adiciona os headers de comunicação.
+
+1.  O [apiserver proxy](/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services):
+
+    - é um bastion server, construído no servidor de API
+    - conecta um usuário fora do cluster com os IPs do cluster que não podem ser acessados de outra forma
+    - executa dentro do processo do servidor de API
+    - cliente para proxi usa HTTPS (ou http se o servidor de API for configurado)
+    - proxy para o destino pode usar HTTP ou HTTPS conforme escolhido pelo proxy usando as informações disponíveis
+    - pode ser usado para acessar um Nó, Pod ou serviço
```

Comment:
The word `Service` here means a Kubernetes Service, so do you not need to translate, can use the original word.

Reply:
Should `Nó`be kept as `node`?

Reply:
The changes referring to `Services`where applied at: ea9e20c513a6759ecd4a9e13869b3d083d5b96d9. Feel free to close and merge in case the word `Nó`should keep unchanged.

Reply:
I've answered your question at the slack xD